### ISSUE TITLE
TypeError calling resource

### ISSUE THREAD

**Issue by jamesjwarren** [jbalboni/super-res#6]
Calling `superRes.resource('/url/:id')` as per your documentation, errors out:

```
TypeError: Cannot convert undefined or null to object
    at Function.getOwnPropertyNames (native)
    at new ResourceAction (.../node_modules/super-res/dist/super-res.js:51:14)
    at generateDefaultActions (.../node_modules/super-res/dist/super-res.js:174:18)
    at Object.super_res__superRes.resource (.../node_modules/super-res/dist/super-res.js:192:20)
    at Object.<anonymous> (.../src/fm.js:5:19)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
```

**Comment by jbalboni** [comment]
Ack, sorry. I wonder if that broke with the last PR. I'll check it out today.

**Comment by jbalboni** [comment]
Pushed a fix to npm as 3.0.1.

**Comment by jamesjwarren** [comment]
Cheers!
